circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -287,12 +303,15 @@ private Copy submitCopyJob(CopyJobRequest copyJob) {
           }
         } catch (AmazonClientException e) {
           CopyObjectRequest copyObjectRequest = copyJob.getCopyJobRequest().getCopyObjectRequest();
-          LOG
-              .info("Copying '{}/{}' failed, adding to retry list.",
-                  copyObjectRequest.getSourceBucketName(),
-                  copyObjectRequest.getSourceKey());
-          LOG
-              .debug("Copy failed with exception:", e);
+
+          if (e.getMessage() == "The provided token has expired.") {
+            LOG.info("S3 copier failed due to token expiration. Not counting failure against this request.");
```

Comment:
What is the significance of this change?  Will it then be re-tried with a new S3 client that got new credentials with a new timeout?  Is this even still needed after the fix to AssumeRoleProvider?